src/smsfusion/_ins.py

```diff
@@ -763,26 +815,23 @@ def update(
         if degrees:
             w_imu = (np.pi / 180.0) * w_imu
 
-        # Update current state estimate
-        self._x = self._ins.x
+        # Current INS state estimates
+        pos_ins = self._ins._pos
+        vel_ins = self._ins._vel
+        q_nm_ins = self._ins._q_nm
+        bias_acc_ins = self._ins._bias_acc
+        bias_gyro_ins = self._ins._bias_gyro
 
-        # Current state estimates
-        pos_ins = self._pos
-        vel_ins = self._vel
-        q_ins_nm = self._q_nm
-        bias_acc_ins = self._bias_acc
-        bias_gyro_ins = self._bias_gyro
-
-        R_ins_nm = _rot_matrix_from_quaternion(q_ins_nm)  # body-to-ned rotation matrix
+        R_nm_ins = _rot_matrix_from_quaternion(q_nm_ins)  # body-to-ned rotation matrix
 
         # Bias compensated IMU measurements
         f_ins = f_imu - bias_acc_ins
         w_ins = w_imu - bias_gyro_ins
 
         # Update system matrices
-        self._update_F(q_ins_nm, f_ins, w_ins)
-        self._update_G(q_ins_nm)
-        self._update_H(q_ins_nm)
+        self._update_F(q_nm_ins, f_ins, w_ins)
```

Comment:
I think I know the answer, but note to self:

Why `f_ins` but not when calculating the ref_g?

Reply:
I was not completely sure about this myself, but after having had a look at the derivations in the Fossen book it seemd to me like we should use `f_ins` and `w_ins` in the system matrices. This is also logical, since the system matrices are obtained by linearizing the system about the INS-state. 
![image](https://github.com/4Subsea/smsfusion-python/assets/68221790/7e4c159f-3388-445d-8b5d-d923ad2f985f)

Reply:
Regarding the ref_g, I thought it would be best to use the "best" estimate of the bias (i.e., including the error-state bias) there